Legacy example cleanup (#21535)
mdn/content: files/en-us/web/css/@supports/index.md

}
```

### Testing for the support of a selector

> The CSS Conditional Rules Level 4 specification adds the ability to test for support of a selector—for example {{cssxref(":has",":has()")}}.

Reviewer: ```suggestion
The CSS Conditional Rules Level 4 specification adds the ability to test for the support of a selector such as  {{cssxref(":has",":has()")}}.
```

Reviewer: Should the word "selector" in the heading and in the paragraph be changed to pseudo-class?

Author: It's not just pseudo classes, it could be pseudo-elements as well and perhaps a combinator.

```css

> /* This rule won't be applied in browsers which don't support :has() */
> ul:has(> li li) {
>   /* CSS applied when the :has(…) selector is supported */

Reviewer: ```suggestion
/* This rule won't be applied in browsers that don't support :has() */
ul:has(> li li) {
  /* CSS is applied when the :has(…) pseudo-class is supported */
```

}

@supports not selector(:has(a, b)) {
  /* Fallback for when :has() is unsupported */
  ul > li,
  ol > li {

>     /* The above expanded for browsers which don't support :has(…) */

Reviewer: ```suggestion
    /* The above expanded for browsers that don't support :has(…) */
```
This part "expanded for browsers" is not clear.
